hydra-node/src/Hydra/Options.hs

```diff
@@ -682,7 +685,7 @@ toArgs
         []
 
     argsChainConfig =
-      ["--network-id", toArgNetworkId networkId]
+      ["--testnet-magic", toArgNetworkId networkId]
```

Comment:
This is incomplete. The `ChainConfig` might want to configure a `Mainnet` network id.. then `--testnet-magic` is not the right option.

Must: Make this generate `--mainnet` when needed.